Javascript/QuizApp/index.js

```diff
@@ -47,41 +16,31 @@ radioButtons.forEach((radioButton) => {
   });
 });
 
-// New Button for Search Bar
-const searchButton = document.createElement("button");
-searchButton.innerText = "Search Questions";
-searchButton.classList.add("control-buttons");
-searchButton.type = "button";
-buttonDiv.appendChild(searchButton);
-searchButton.addEventListener("click", searchInQuestions);
-searchButtonCreated = true;
+//Input Check Applied to check length of Question
+const questionInputCheck = document.getElementById("question");
+const maxLengthOfInput = questionInputCheck.maxLength;
+question.addEventListener("input", (event) => {
+  const currentLength = event.target.value.length;
+  if (currentLength >= maxLengthOfInput) {
+    errorOrNormalMessageContainer(
+      "ERROR: Question cannot be more than 140 characters!\nThis app saves only first 140 characters of input question."
```

Comment:
```suggestion
      `ERROR: Question cannot be more than ${maxLengthOfInput} characters!\nThis app saves only first ${maxLengthOfInput} characters of input question.`
```

It was a good idea to get the `maxLength` attribute of the input instead of a hardcoded number. Let's use the `maxLength` everywhere.